test/markups/README.asciidoc.html

```diff
@@ -16,7 +16,7 @@
 <p>Refer to <a href="#another-section">Another Section</a> or <a href="#another-section-1">Another Section</a>.</p>
 </div>
 <div>
-<p>Navigate to <a href="another-document.adoc">another document</a>.</p>
+<p>Navigate from README.asciidoc to <a href="another-document.asciidoc">another document</a>.</p>
```

Comment:
@kivikakk notice that the extension of referenced file now matches the extension of the file being converted.